### PR TITLE
33 charindex issue

### PR DESCRIPTION
**Pull request by yafimvo**
We use `parse_argstring` from `IPython.core.magic_arguments` to parse line queries (`%sql`).
This parser was designed to parse shell-like args, like:

```
{program} {command} arg1 arg2 -a --something value
```

The problem is that users might pass an embedded SQL query:

```
{program} {command} arg1 arg2 -a --something value {some-sql-query-here}
```
And the query might contain spaces, leading to errors in parsing.

## The fix
In order to avoid overriding the parsing logic, If the parsing fails or only a line without args (`%sql`) was given - we try to execute the line as if it was a cell.

original issue: #33

### CODE REVIEW

File: src/tests/test_command.py [hunk 1]
```diff
@@ -5,12 +5,41 @@
...
+@pytest.mark.parametrize(
+    "line, query_result",
+    [
+        ("SELECT instr(' ', ' ')", [1]),
+        ("SELECT instr(' a ', ' ')", [1]),
+        ("SELECT instr('aaa', ' ')", [0]),
+        ("SELECT instr('aaa', 'a')", [1]),
+        ("SELECT COUNT(*) FROM test", [2]),
+        ("SELECT * FROM test where n == 1", [(1, 'foo')]),
+        ("SELECT * FROM test", [(1, 'foo'), (2, 'bar')])
```
Comment (edublancas) [inline src/tests/test_command.py:25]: can you add some arguments after and before the query? I think that's what will break this

**Review by edublancas — CHANGES_REQUESTED** [review]

File: src/tests/test_command.py (hunk 1 above)
Comment (yafimvo) [reply src/tests/test_command.py:25]: Yes, you are right.
running the following command will return an empty result. ([])
```
%sql --save result SELECT instr(' ', ' ')
``` 

since the cell expects the following structure

```
%%sql --save result
SELECT instr(' ', ' ')
```

If I run this command without this fix it will throw `ValueError: No closing quotation` which I'm not sure what is better.

I can catch this exception and suggest the user use `%%sql` instead.
Another quick option that might be a base for the future parser. We can hold the list of possible arguments and build the cell based on this list and the given line. In that case, we'll need to maintain this list of arguments.

Comment (idomic) [reply src/tests/test_command.py:25]: Right, the second approach is a kind of heuristics to capture %s of the cases (let's say 70%). 
I think the value error is a bug in its own right?

Comment (edublancas) [reply src/tests/test_command.py:25]: This is proving too complex for an edge case that only a few people will have. I suggest closing this.

Comment (idomic) [reply src/tests/test_command.py:25]: I agree
